services/portfolios_ca/src/services/targets.ts

```diff
@@ -57,7 +103,7 @@ export default () => {
      *
      * @returns {Object[]} List of targets with portfolios values
      */
-    getTargetsList: async () => [
+    getTargetsList: async (): Promise<object[]> => [
```

Comment:
it's not necessary to describe this output, it's already covered in the service type.

But please change the type in the doc (line 104)

Reply:
Solved